js/activity.js

```diff
@@ -3098,9 +3098,11 @@ function Activity() {
                     [jsONON.length,"hidden",0,0,[val, last ? null: jsONON.length +1]]                  
                 );
             }
-            console.debug(JSON.stringify(jsONON));
-            console.debug ('midi track loading ... be patient ');
+            //console.debug(JSON.stringify(jsONON));
             console.debug ('finished when you see: "block loading finished "');
+            document.body.style.cursor = "wait";
+            logo.textMsg ('midi tracks loading');
```

Comment:
We should combine this into one message (and enable translation):

logo.textMsg(_("MIDI loading. This may take some time depending upon the number of notes in the track"));